CMakeLists.txt

```diff
@@ -46,7 +46,7 @@ option(WarpX_ASCENT        "Ascent in situ diagnostics"                 OFF)
 option(WarpX_MPI           "Multi-node support (message-passing)"       ON)
 option(WarpX_OPENPMD       "openPMD I/O (HDF5, ADIOS)"                  OFF)
 option(WarpX_PSATD         "spectral solver support"                    OFF)
-option(WarpX_QED           "PICSAR QED (requires Boost and PICSAR)"     OFF)
```

Comment:
I know this sounds a bit like a roll backwards, but we could try to switch this option to default-`ON` for a while before removing it completely.

The `#idef` guards in the code are not too intrusive and it would allow to control the dependency in some cases, i.e. exotic new architectures and testing done by external collaborators that need a very small subset of WarpX. Just an idea.

Reply:
Sure, we can do that

Reply:
I will open a new PR, since it is probably easier to start from scratch.